x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/action/privilege/GetPrivilegesRequestBuilder.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+package org.elasticsearch.xpack.core.security.action.privilege;
+
+import org.elasticsearch.action.ActionRequestBuilder;
+import org.elasticsearch.client.ElasticsearchClient;
+
+/**
+ * Builder for {@link GetPrivilegesRequest}
+ */
+public class GetPrivilegesRequestBuilder
+        extends ActionRequestBuilder<GetPrivilegesRequest, GetPrivilegesResponse, GetPrivilegesRequestBuilder> {
+
+    public GetPrivilegesRequestBuilder(ElasticsearchClient client) {
+        this(client, GetPrivilegesAction.INSTANCE);
+    }
+
+    GetPrivilegesRequestBuilder(ElasticsearchClient client, GetPrivilegesAction action) {
```

Comment:
same thing as the delete action. Lets make this one public and remove the constructor above it. In the client code we can use the action instance